wp-menu-cart.php

```diff
@@ -21,9 +21,12 @@
 
 class WpMenuCart {	 
 
-	protected $plugin_version = '2.10.4';
+	protected     $plugin_version         = '2.10.4';
 	public static $plugin_slug;
 	public static $plugin_basename;
+	public static $is_block_editor        = false; // In the Full Site Editor the menu item is loaded inside the block editor iframe, traditional WP functions aren't available there.
```

Comment:
I don't think this should be a class property. Instead, you can create a class method `is_rendering_block()` which checks:
```php
if ( defined( 'REST_REQUEST' ) && REST_REQUEST ) {
	$route = untrailingslashit( $GLOBALS['wp']->query_vars['rest_route'] );
	if ( strpos( $route, 'wpo/wpmenucart-navigation' ) !== false ) {
		return true;
	}
}
return false;
```
This is much more accurate and doesn't require setting and unsetting a property.

Reply:
The issue is that i need to access it from `/wp-menu-cart/includes/wpmenucart-woocommerce.php` to maybe load this:

```
$this->maybe_load_session();  // in backend (full site editor) the wc session is null
$this->maybe_load_customer();
```